src/chapters/cleaning-data/exercises.rst

```diff
@@ -4,23 +4,27 @@ Exercises: Cleaning Data
 Getting Started
 ---------------
 
-Fork this `GitHub repository <https://github.com/launchcodeeducation/cleaning-data/blob/main/Cleaning%20Data%20Exercises.ipynb>`__ and 
-clone to your computer. If you need a refresher on how to do this 
-see :ref:`Instruction for Using GitHub w/Jupyter Notebooks<usingGitHubNotebooks>`.
+Fork this `GitHub repository <https://github.com/launchcodeeducation/cleaning-data/blob/main/Cleaning%20Data%20Exercises.ipynb>`__ and clone to your computer. 
 
-| Next, select one of the following data sets to work with.  Download and add to your project folder in Jupyter Notebooks.
+If you need a refresher on how to do this see :ref:`Instruction for Using GitHub w/Jupyter Notebooks<usingGitHubNotebooks>`.
+
+Code Along
+----------
+
+Select one of the following data sets to work with.
 
 #. `Women's E-commerce Clothing Reviews <https://www.kaggle.com/nicapotato/womens-ecommerce-clothing-reviews>`__.
 #. `New York City Rat Sightings <https://www.kaggle.com/new-york-city/nyc-rat-sightings>`__.
- 
-| Work through the exercise notebook using the dataset you have choose from above.
 
-| Don't hesitate to reach out to course staff and your fellow students if you want to talk through the 
-  data set or a potential spot that needs to be cleaned!
+Download and add to your Jupyter Notebook.
+
+Work through the exercise notebook using the dataset you have choose from above.
```

Comment:
Suggestion - Think a list within a list would be easier to read.

#. Select one of the following data sets to work with.

   #. `Women's E-commerce Clothing Reviews <https://www.kaggle.com/nicapotato/womens-ecommerce-clothing-reviews>`__.
   #. `New York City Rat Sightings <https://www.kaggle.com/new-york-city/nyc-rat-sightings>`__.

#. Download and add to your Jupyter Notebook.

#. Work through the exercise notebook using the dataset you have choose from above.